### DISCUSSION TITLE
Brush Set export: Can it be developed or will it not be needed?

### DISCUSSION THREAD

**Discussion by ghost** [photopea/photopea#7198] [Ideas]
I understand that photopea was made to be a basic (but powerful) application. But would it be possible to develop the resources to export the brushes in sets? (exporting my brushes that I created all in an .ABR file). because I'm that kind of person who likes to make art and create brush presets. And yes, I know there is a way to export it to a .TPL file, but it doesn't save the brush texture. and if I can't develop this feature, maybe I don't need to export just individual brushes

**Comment by photopea** [comment]
How exactly can I help you? You can export brushes as an ABR file in Photopea, what exactly is not working?

**Reply by ghost** [reply]
everything is working. but I wanted to know how to combine the brushes into a single ABR file

**Reply by photopea** [reply]
I am confused, what do you mean by combining brushes?

**Reply by ghost** [reply]
I want to say, add brushes. for example: 20 brushes in an ABR file

**Reply by ghost** [reply]
a pack of brushes

**Reply by ghost** [reply]
I don't want to export just one, I want a package

**Reply by photopea** [reply]
If you press "Export as .ABR" in Photopea, it will save all your brushes as one ABR file. It is described here: https://www.photopea.com/learn/brush-tools#:~:text=along%20the%20stroke-,Brush%20Presets,-Photopea%20supports%20loading

**Reply by ghost** [reply]
Are you sure? because I exported it, analyzed it and only one brush came out

**Reply by photopea** [reply]
Please, describe exactly what you are doing. What do you do after opening www.photopea.com to get the ABR file with one brush?

**Reply by ghost** [reply]
I just clicked normally to export the ABR file. so unfortunately only export just one

**Reply by photopea** [reply]
you can select the brushes that you want to export by holding the Shift key.

**Reply by ghost** [reply]
my God! it worked! thanks
